filechanges/opt/musicbox/startup.sh

```diff
@@ -224,13 +224,26 @@ fi
 # renice mopidy to 19, to have less stutter when playing tracks from spotify (at the start of a track)
 renice 19 `pgrep mopidy`
 
-if [ "$INI__musicbox__autoplay" -a "$INI__musicbox__autoplaywait" ]
+if [ "$INI__musicbox__autoplay" -a "$INI__musicbox__autoplaymaxwait" ]
```

Comment:
I know we didn't have this previously but is it worth adding a bit of validation to `$INI__musicbox__autoplaymaxwait` while we are here? Maybe checking it's digits or something?